### ISSUE TITLE
[BUG] a11y GC theme - <nav> missing aria-label

### ISSUE THREAD

**Issue by GoCHacpie** [wet-boew/wet-boew#9411]
## Required information

**Describe the bug**
The <nav> for skip link section is missing aria-label
The <nav> for breadcrumbs is missing aria-label

**To Reproduce**
1. View code and look for <nav> tags

**Expected behavior**
The <nav> for skip link section should have aria-label e.g. <nav aria-label="Skip links"> 
aria-label="Saut de navigation"
The <nav> for breadcrumbs should have aria-label e.g The <nav aria-label="Breadcrumb">  FR: <nav aria-label="Fil d'Ariane">
https://www.w3.org/WAI/ARIA/apg/example-index/breadcrumb/index.html

**Who shall do the work?**
Not sure

## Additional information (optional)

**Version of WET-BOEW/GCWeb you are using**
current version

**Desktop/Smartphone (please complete the following information)**
All browsers

**Screenshots**
location in code
![image](https://user-images.githubusercontent.com/74194190/184962215-94cb3164-b469-4aa2-80db-ef5f9463dac2.png)
![image](https://user-images.githubusercontent.com/74194190/184962294-111c9d27-5315-47ba-800c-607e315a22bf.png)

**Additional context**
This is accessibility best practice.
https://www.w3.org/WAI/ARIA/apg/example-index/breadcrumb/index.html

**Comment by duboisp** [comment]
This request can fit in both repository.

I do remember that @shawnthompson has proposed something similar and by recommendation was to automate it, via JS, based on the top heading of the `<nav>` section.

I am going to add this feature request at our WCAG upgrade project

**Comment by shawnthompson** [comment]
Related: #9141
